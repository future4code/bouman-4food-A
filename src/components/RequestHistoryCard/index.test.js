import React from 'react';
import {RequestHistoryCard} from './index';
import renderer from "react-test-renderer";


// it('footer renders correctly', () => {
    
//         const tree = renderer
//             .create(<RequestHistoryCard/>)
//             .toJSON();
//         expect(tree).toMatchSnapshot();
    
// })

