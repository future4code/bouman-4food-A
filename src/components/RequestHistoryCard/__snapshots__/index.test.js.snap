// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders correctly 1`] = `
<div
  className="sc-bdVaJa eeaEId"
>
  <div
    className="MuiPaper-root sc-ifAKCX gQakHq MuiPaper-elevation1 MuiPaper-rounded"
  >
    <p
      className="sc-bwzfXH jbnfSX"
    />
    <p
      className="sc-htpNat bLgUvi"
    />
    <p
      className="sc-bxivhb cAIVKn"
    />
  </div>
</div>
`;
